tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -612,6 +612,12 @@ internal static class FeatureFlags
             /// Default value is <c>false</c> (disabled).
             /// </summary>
             public const string TraceId128BitLoggingEnabled = "DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED";
+
+            /// <summary>
+            /// Configuration key to enabling or disabling the collection of shell commands executions.
+            /// Default value is <c>false</c> (disabled).
+            /// </summary>
+            internal const string CommandsCollectionEnabled = "DD_COMMANDS_COLLECTION_ENABLED";
```

Comment:
This should begin with `DD_TRACE_`, and I think we should add `SHELL` because "commands" it too ambiguous. Maybe `DD_TRACE_SHELL_COMMANDS_COLLECTION_ENABLED`? It's a bit long, so if you have other suggestions...